### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by blainerothrock** [blainerothrock/earthquakes#1]
Install and successful run pre-processing, training and testing scripts from deep metric learning model

**Comment by blainerothrock** [comment]
## Installation Instructions
* `conda create -n earthquakes --python=3.7`
* `conda install numpy` <-- must be install before `obspy` dependency in `requirements.txt`
* Removed version dependency for `scikit-learn` and `matplotlib`. Using latest versions
* `pip install -r requirements.txt`

**Comment by blainerothrock** [comment]
## Data
* The data to reproduce the paper results can be found [here](https://northwestern.app.box.com/folder/109657627928?utm_source=trans&utm_medium=email&utm_campaign=collab%2Bauto%20accept%20user) (NW Box account)

Tremor data is found on the `GPGPU` machine.

**Comment by blainerothrock** [comment]
## PyCharm setup

I cloned the repo on my Ubuntu machine and using PyCharm's deployment function synced the repo to my local machine. All data lives on the remote server, but I can edit locally on my mac.

**Comment by blainerothrock** [comment]
* Running the provided scripts from the `README` seem to produce desired results... although when testing the prediction do not seem to match.
